components/Utils/ActionBubble.js

```diff
@@ -155,9 +156,7 @@ const ActionBubble = ({ addresses, bounty, action }) => {
       <div className='w-full flex-0 rounded-sm overflow-hidden ml-4 border-web-gray border-b before:w-2 before:h-2 before:bg-nav-bg before:absolute before:absolute before:left-12 before:top-[34px] before:border-b  before:border-l before:border-web-gray before:rotate-45  border'>
         <div className={`bg-nav-bg w-full pl-3 ${!action && 'border-web-gray'} flex justify-between`}>
           <span className='py-2'>
-            <span data-testid='actionTitle'>
-              {tokenValues || (!action?.receiveTime && !action?.refundTime) ? titlePartOne : <Skeleton width='34' />}
-            </span>
+            <span data-testid='actionTitle'>{tokenValues || titlePartOne}</span>
```

Comment:
Token values should not be used as a possible value for this one as its an obj.

Reply:
just `{titlePartOne}` then?  
that's the bit I was unsure of, was wondering why there was this conditional initially (`(!action?.receiveTime && !action?.refundTime) ?`)

Reply:
Just `titlePartOne`
Not exactly sure. my best guess would be it was  so that the skeleton wouldn't show on minted deposits when the graph hadn't loaded yet. Should be find to remove though.